packages/core/src/fetchers/fetchExperienceEntry.ts

```diff
@@ -3,7 +3,9 @@ import { ContentfulClientApi, Entry } from 'contentful';
 type fetchExperienceEntryArgs = {
```

Comment:
Now we're exposing these methods to the developers I think we should make sure each of the parameters and the function itself include a doc string so that they don't have to reference external docs to understand what each function does.

Reply:
Good point, I added those now directly on the function as the types per parameter are not readable for SDK users.